src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  padding-top: 0;
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.is-active {
+  font-weight: 500;
+  color: var(--blue-color);
+  position: relative;
+  font-size: 12px;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+}
```

Comment:
The `.is-active` class is styled correctly, but ensure that its styles do not conflict with those of `.nav_link`. Consider merging common styles to avoid redundancy.